ocs_ci/helpers/helpers.py

```diff
@@ -1377,6 +1447,47 @@ def get_provision_time(interface, pvc_name, status="start"):
     return datetime.datetime.strptime(stat, DATE_TIME_FORMAT)
 
 
+def get_pvc_bulk_deletion_time(interface, pv_names, status="start"):
+    """
+    Get the deletion time of PVC bulk based on csi provisioner logs
+    Args:
+        interface (str): The interface backed the PVC
+        pv_names (list): Names of the PVs which are backend objects for PVCs deleted
+        status (str): the status that we want to get - start / end time of bulk deletion
+    Returns:
+        datetime object: The time of bulk pvc deletion
+    """
+
+    # Define the operation that need to retrieve
+    operation = "started"
+    if status.lower() == "end":
+        operation = "succeeded"
+
+    this_year = str(datetime.datetime.now().year)
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_csi_provisioner_pod(interface)
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(pod_name[0], "csi-provisioner")
+    logs += pod.get_pod_logs(pod_name[1], "csi-provisioner")
+
+    logs = logs.split("\n")
+
+    all_stats = []
+    for pv_name in pv_names:
+        stat = [i for i in logs if re.search(f'delete "{pv_name}".*{operation}', i)]
+        mon_day = " ".join(stat[0].split(" ")[0:2])
+        stat = f"{this_year} {mon_day}"
```

Comment:
We need to be more efficient here. Looking through PVC deletion start and end time for 720 volumes means, parsing through the same log 1440 times. Maybe there is a better way?

Reply:
@keesturam - I've tried to think about it and I don't see how we find the relevant records in the log more efficiently.

Reply:
Don't we have similar function in performance or helpers ?, it's good to enhance these functions.
measure_pv_deletion_time_bulk(interface, pv_name_list, wait_time=60) -- https://github.com/red-hat-storage/ocs-ci/blob/abcced1c32c2341449a00612e27188fca56f86ca/ocs_ci/helpers/helpers.py#L1520

Reply:
@ramkiperiy  Yes, we do have this function, but it measured different time , not what we need. 
 def measure_pv_deletion_time_bulk provides a dictionary with deletion time for EACH one of the PVCs in a bulk and the bulk_deletion in this PR is meant to measure deletion time starting from the start of the deletion of the first PVC and till the end of the deletion of the last deleted PVC. 

I started with trying to customise the existing code ( measure_pv_deletion_time_bulk) , but it became cumbersome. 
Therefore, even though I realize that part of the code is duplicated, I chose to implement it in a ceparate function.